### PR TITLE
working speed Indicator

### PR DESCRIPTION
**Pull request by kzverev**
Code Review

### CODE REVIEW

**Comment by marcopeg** [comment]
How did you run the component?
I see there is no examples in the styleguide.

File: app/client/components/SpeedIndicator.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import React from 'react';
+
+export default class SpeedIndicator extends React.Component {
+
+    static propTypes = {
+        speed: React.PropTypes.number.isRequired,
+    }
+
+    static defaultProps = {
+        speed: 'wer',
```
Comment (marcopeg) [inline app/client/components/SpeedIndicator.js]: you have defined the property `speed` as a _required_ numeric input so:
1. you don't need a default value 'cause is required
2. even if you default it you should default to a number

File: app/client/components/SpeedIndicator.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    }
+
+    render() {
+        var { speed } = this.props;
+
+        var speedStyle = {
+          backgroundColor: 'red',
+          width: '30px',
```
Comment (marcopeg) [inline app/client/components/SpeedIndicator.js]: when it come to `px` you should just give in a number and React will do the magic:

```
var stuyle = {
  width: 22,
};
```

**Comment by kzverev** [comment]
changes are committed

**Comment by marcopeg** [comment]
I see an improvement and I think you can push it a bit further:
the styleguide should display all the possible combinations of properties.

In this case you should provide an example to display each possible generated color.

The objective of this is that a designer should be able to run the styleguide and have a comprehensive understanding of your front end job.

**Comment by kzverev** [comment]
Styleguide was expanded with examples.

**Comment by marcopeg** [comment]
👍

**Pull request merged**